.circleci/config.yml

```diff
@@ -336,6 +336,32 @@ commands:
             RELIER_DOMAIN: << pipeline.parameters.relier-domain >>
             UNTRUSTED_RELIER_DOMAIN: << pipeline.parameters.untrusted-relier-domain >>
 
+  run-playwright-tests-chromium:
+    parameters:
+      project:
+        type: string
+    steps:
+      - run:
+          name: Running Playwright tests
+          # Supports 'Re-run failed tests only'. See this for more info: https://circleci.com/docs/rerun-failed-tests-only/
+          command: |
+            npx nx build fxa-auth-client
+            cd packages/functional-tests/tests
+            TEST_FILES=$(circleci tests glob "./**/*.spec.ts")
+            cd ..
+            echo $TEST_FILES | circleci tests run --command="xargs yarn playwright test --project=<< parameters.project >>-chromium $GREP" --verbose --split-by=timings --timings-type=classname
+          environment:
+            NODE_OPTIONS: --dns-result-order=ipv4first
+            JEST_JUNIT_OUTPUT_DIR: ./artifacts/tests
+            JEST_JUNIT_ADD_FILE_ATTRIBUTE: true
+            PLAYWRIGHT_BLOB_OUTPUT_DIR: ./artifacts/blob-report
```

Comment:
it will create separate `xml` and `blob reports` for `Firefox` and `Chromium`
<img width="814" alt="Screenshot 2024-09-20 at 10 54 46 AM" src="https://github.com/user-attachments/assets/0cc902d2-1623-4a55-bbd5-18f0de9bac85">

<img width="789" alt="Screenshot 2024-09-20 at 10 54 29 AM" src="https://github.com/user-attachments/assets/8905b22f-b909-4efd-bad1-24357f6cb286">